### ISSUE TITLE
Track Time Worked On Project

### ISSUE THREAD

**Issue by AnMcCbusiness** [woodrdk/CarRepairTracker#35]
- Download Timesheet -Time Tracker app on PlayStore
- Update this issue with time spent worked on app after done working

**Comment by AnMcCbusiness** [comment]
Initial hours before App for Anthony:
- 4 hours

**Comment by woodrdk** [comment]
Hours before App unknown .

should we also track what we did when we update here?

**Comment by AnMcCbusiness** [comment]
I'd probably put you at least 80 hours already, ha!

**Comment by AnMcCbusiness** [comment]
Yeah, what we did on individual branches or merges would be a good keep-track, too. Just something to submit when he wants the info.

**Comment by woodrdk** [comment]
Will post a list of prior to 8/1/2018 this Friday 8/3/2018.

For the evening of 8/1 I spent 1 hour and 12 minutes according to the Timesheet -Time Tracker app.
I was cleaning up and adding to the database and adding migration along with updating database.

**Comment by AnMcCbusiness** [comment]
8/2/18

- Two hours troubleshooting with Rob talking with instructor and higher level students about passing objects and setting them against a Parent MDI Form

**Comment by AnMcCbusiness** [comment]
8/6/18
- One hour fixing and troubleshooting with Rob fixing the delete user from database form and code
